projects/schematics/yarn.lock

```diff
@@ -0,0 +1,3836 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We no longer should have yarn.lock in libraries. All dependencies used in them should come from root package.json and node_modules

Reply:
Removed.